src/ViewModel/ContentHeaderNew.php

```diff
@@ -49,7 +50,8 @@ public function __construct(
         MetaNew $meta = null,
         Doi $doi = null,
         string $licence = null,
-        AudioPlayer $audioPlayer = null
+        AudioPlayer $audioPlayer = null,
+        bool $hasAside = false
```

Comment:
We need to amend the ContentHeaderNewTest

Reply:
Done. The tests cover now hasAside.